src/content/docs/new-relic-solutions/overview.mdx

```diff
@@ -3,23 +3,22 @@ title: Solutions, guides, and best practices
 type: landingPage
 tags:
   - New Relic solutions
-metaDescription: New Relic best practice guides and solutions landing page.
+metaDescription: An overview of New Relic guides for helping you optimize your monitoring and observability practices.  
 redirects:
   - /docs/solutions-best-practices-landing-page
   - /docs/using-new-relic/welcome-new-relic/measure-devops-success/guide-measuring-devops-success
   - /docs/new-relic-solutions
   - /docs/new-relic-solutions/table-of-contents
 ---
 
-You need to know exactly how your software performs so you can identify issues, resolve them quickly, and track improvements. And you want to learn from each iteration so your software, customer experience, and business outcomes become even better.
+Our guides give you best practices and tips for optimizing how you use New Relic. If you don't yet have New Relic and have questions, [contact New Relic sales](https://newrelic.com/contact-sales). 
 
-Our guides provide a foundation you can build on and customize based on your goals. During each step of your journey, we can help you with recommended key performance indicators (KPIs), dashboards, how-to docs, and more. If you're not yet a New Relic customer and you have questions, [contact New Relic sales](https://newrelic.com/contact-sales). 
-
-Using our proven methodology, recommendations, and expert guidance, your organization can move faster, stay on course, and transform successfully! Get started with our guides: 
+Here's an overview of our guides: 
 
 * **Set up New Relic**: see [our Install doc](/docs/new-relic-solutions/new-relic-one/install-configure/install-new-relic) for options.
-* **Learn about the platform**: want to understand common platform experiences and features? See [our platform UI docs](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). 
-* **Best practices guides**: [Learn best practices for some key New Relic features](/docs/new-relic-solutions/best-practices-guides).
-* **Observability maturity**: [Our observability maturity series](/docs/new-relic-solutions/observability-maturity/introduction) identifies common observability goals and teaches you how to meet them. 
+* **Learn about the platform**: want to understand common platform experiences and features? [Get to know the platform.](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one)
```

Comment:
```suggestion
* **Learn about the platform**: want to understand common platform experiences and features? [Get to know the platform](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one).
```

Reply:
i could be wrong on this, but when I do a full sentence link I include the period. I don't often do full sentences as links but that's what I do in that case.

Reply:
Well the other bullet points have the period outside of the hyperlink, so maybe just be consistent about it?